src/incflo.H

```diff
@@ -499,7 +518,50 @@ private:
     Real tau_0 = 0.0;
     Real papa_reg = 0.0;
     Real eta_0 = 0.0;
-
+    
+    //////////////////////////////////////////////////////////////////////////////////////////////
+    //
+    // ABL Physics
+    //
+    //////////////////////////////////////////////////////////////////////////////////////////////
+    
+    bool coriolis_effect = false;
+    bool abl_forcing = false;
+    bool sgs_model = false;
+    
+    Vector<Real> temperature_heights;
+    Vector<Real> temperature_values;
+    int ntemperature = 1;
+    
+    Real cutoff_height = 600.0;
+    Real Uperiods = 4.0;
+    Real Vperiods = 4.0;
+    Real deltaU = 1.0;
+    Real deltaV = 1.0;
+    Real zRefHeight = 50.0;
+    Real theta_amplitude = 0.8;
+    Real abl_forcing_height = 70.0;
+    Real kappa = 0.4; //von Karman constant
+    Real surface_roughness_z0 = 0.0001; // surface roughness [m]
```

Comment:
Did you mean a static struct or a const struct? Because the former isn't any better than the current form. I would like to see capabilities like ABL physics abstracted into classes that expose the relevant terms and are separate from the giant incflo member variables.